examples/visual-tests/amp-by-example/components/amp-form/embed/index.html

```diff
@@ -840,7 +840,7 @@ <h3 class="abe-code p1 caps h4">Example</h3>
           <h2 id="other-form-input-samples" class="www-heading pb4 mb2 relative h3">Other form input samples</h2>
         </section>
       <section class="www-component-desc">
-          <p class="mb2 px1">  <code>amp-form</code> supports all HTML5 form types with the exception of file, password and image.
+          <p class="mb2 px1">  <code>amp-form</code> supports all HTML5 form types with the exception of file, button and image.
```

Comment:
I don't think we should be changing these. @rsimha are these auto scrapped from ABE once in a while?

Reply:
Good to know. Thanks!

Reply:
I just checked: There are a handful of new / renamed components, so it's time to update the example pages.

Reply:
To move forward, should I do that as part of this PR, or is it ok to do that in a follow up? Should I keep the manual changes to these files in this PR?

Reply:
The manual changes here will get squashed by running the script, so I'd leave them out of here.

If you want to update the example files before merging this PR (so the tests use the new version), it should be a fairly quick process to run build-system/copy-abe-visual-test-pages.sh. Let me know if you run into any problems.

Reply:
I'll update these in a follow-up PR